### DISCUSSION TITLE
عدم نمایش بات تلگرام در منو

### DISCUSSION THREAD

**Discussion by ne7worker** [hiddify/Hiddify-Manager#362] [General سایر]
سلام، وقتتون بخیر. من به نسخه 1.5.3 آپدیت کردم ولی بات تلگرام به منو اضافه نشد

**Comment by raminrez** [comment]
اول نیازه که توکن تلگرام داخل تنضیمات اضافه بشه
بعد از اعمال تغییرات اضافه میشه
